### PR TITLE
fix(pci.projects.new): fix creation error message

### PR DESCRIPTION
**Pull request by Jisay**

### CODE REVIEW

File: packages/manager/modules/pci/src/error/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -7,9 +7,9 @@
...
-  "pci_error_paypal_account_not_verified": "Malheureusement vous ne pouvez pas créer de projet Public Cloud avec votre compte PayPal car celui-ci n'est pas vérifié. Nous vous invitons à faire les démarches nécessaires auprès de PayPal pour vérifier votre compte, puis de réessayer. Vous pouvez aussi modifier <a href=\"{{ href }}\">votre moyen de paiement par défaut</a>.",
-  "pci_error_unpaid_debts": "Malheureusement vous ne pouvez pas créer de projet Public Cloud car vous avez une dette en cours chez OVH. Nous vous invitons à régler le montant dû, puis de réessayer.",
-  "pci_error_invalid_payment_mean": "Malheureusement vous ne pouvez pas créer de projet Public Cloud car vous n'avez pas de moyen de paiement par défaut valide. Nous vous invitons à modifier <a href=\"{{ href }}\">votre moyen de paiement par défaut</a>."
+  "pci_error_account_not_eligible": "La création d’un nouveau projet Public Cloud n’est pas possible avec votre compte. Nous vous invitons à contacter notre support pour plus d’informations.",
```
Comment (jleveugle) [inline packages/manager/modules/pci/src/error/translations/Messages_fr_FR.json]:
```suggestion
  "pci_error_account_not_eligible": "La création d’un nouveau projet Public Cloud est impossible avec votre compte. Nous vous invitons à contacter notre support pour plus d’informations.",
```

File: packages/manager/modules/pci/src/projects/new/new.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="pci-projects-new h-100">
-    <div class="container-fluid h-100">
-        <div class="row h-100 justify-content-md-center">
-            <div class="col-md-7 p-5 bg-white h-100">
+    <div class="container-fluid overflow-auto h-100">
+        <div class="row minh-100 justify-content-md-center">
```
Comment (jleveugle) [inline packages/manager/modules/pci/src/projects/new/new.html]: If you do that, i think the navbar isn't sticky anymore, right ?

Comment (Jisay) [reply packages/manager/modules/pci/src/projects/new/new.html]: Nope, it's okay with that ;-)

File: packages/manager/modules/pci/src/error/translations/Messages_fr_FR.json (hunk 1 above)
Comment (Jisay) [reply packages/manager/modules/pci/src/error/translations/Messages_fr_FR.json]: This has been validated by content team

File: packages/manager/modules/pci/src/index.scss [hunk 3]
```diff
@@ -4,4 +4,12 @@
   @import '~bootstrap4/scss/_mixins';
   @import '~bootstrap4/scss/_utilities';
   @import '~bootstrap4/scss/_grid';
+
+  .minh-100 {
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/index.scss]: Don't we have a utility CSS package like Bootstrap does ?

Comment (Jisay) [reply packages/manager/modules/pci/src/index.scss]: Nope. There is `mh-100` but it's for `max-height` :-$
And for the class `overflow-auto`, it exists in bootstrap but in v4.3 ;-)

Comment (FredericEspiau) [reply packages/manager/modules/pci/src/index.scss]: Maybe we should have one then, so we don't have to rely on Boostrap ;)

Comment (Jisay) [reply packages/manager/modules/pci/src/index.scss]: But bootstrap is great! :heart: :smile:

File: packages/manager/modules/pci/src/projects/new/new.html (hunk 2 above)
Comment (jleveugle) [reply packages/manager/modules/pci/src/projects/new/new.html]: I think we don't need that because develop has been fixed in another way :)

Comment (Jisay) [reply packages/manager/modules/pci/src/projects/new/new.html]: Nope, the white background is not extended if there is an overflow.
I have merged this branch into an other one, to keep everything at one plae. I close this PR.

**Pull request closed without merging**